qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/WarmupIT.java

```diff
@@ -46,6 +46,7 @@ public void testKNNWarmupDefaultLegacyFieldMapping() throws Exception {
 
     // Custom Legacy Field Mapping
     // space_type : "innerproduct", engine : "nmslib", m : 2, ef_construction : 2
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/k-NN/issues/2415")
```

Comment:
I think you would have a conflict with this. This change is already merged in to `2.x`